src/_utils/_git_secret_tools_linux.sh

```diff
@@ -16,3 +16,11 @@ function __temp_file_linux {
 function __sha256_linux {
   sha256sum "$1"
 }
+
```

Comment:
@sobolevn @simbo1905 

I'm welcoming input on the way to implement the -c option and get the 'permission string' to use for chmod? 

* I figured octal was the best representation to use. 
Agreed?

* I couldn't find just one incantation that would return the octal permissions on OSX and linux, so I'm using `stat --format '%a'` on linux and `stat -f '%p'` on osx. 
Let me know if you see a better way.